virtwho/parser.py

```diff
@@ -310,15 +310,15 @@ def parse_cli_arguments():
     parser.add_argument("-c", "--config", action="append", dest="configs", default=[],
                         help="Configuration file that will be processed, can be used multiple times")
     parser.add_argument("-m", "--log-per-config", action="store_true", dest="log_per_config", default=NotSetSentinel(),
-                        help="Write one log file per configured virtualization backend.\n"
+                        help="[Deprecated] Write one log file per configured virtualization backend.\n"
                              "Implies a log_dir of %s/virtwho (Default: all messages are written to a single log file)"
                              % log.DEFAULT_LOG_DIR)
     parser.add_argument("-l", "--log-dir", action="store", dest="log_dir", default=log.DEFAULT_LOG_DIR,
-                        help="The absolute path of the directory to log to. (Default '%s')" % log.DEFAULT_LOG_DIR)
+                        help="[Deprecated] The absolute path of the directory to log to. (Default '%s')" % log.DEFAULT_LOG_DIR)
     parser.add_argument("-f", "--log-file", action="store", dest="log_file", default=log.DEFAULT_LOG_FILE,
-                        help="The file name to write logs to. (Default '%s')" % log.DEFAULT_LOG_FILE)
+                        help="[Deprecated] The file name to write logs to. (Default '%s')" % log.DEFAULT_LOG_FILE)
     parser.add_argument("-r", "--reporter-id", action="store", dest="reporter_id", default=NotSetSentinel(),
-                        help="Label host/guest associations obtained by this instance of virt-who with the provided id.")
+                        help="[Deprecated] Label host/guest associations obtained by this instance of virt-who with the provided id.")
 
     virt_group = parser.add_argument_group(
```

Comment:
All options in the virt_group as well as satellite_group will be deprecated. (as we are moving to configuration file only).